alarms/alarms.py

```diff
@@ -271,17 +303,20 @@ def _raise_alarm(self, alarm, check_string, arguments):
         last_alarm_time = self._get_time_of_last_alarm(alarm['id'])
 
         # Alarm is not inhibited by no_repeat_interval
-        if last_alarm_time is None or time.time() - last_alarm_time > alarm['no_repeat_interval']:
+        if (
+                last_alarm_time is None or
+                time.time() - last_alarm_time > alarm['no_repeat_interval']
+        ):
             if last_alarm_time is None:
                 _LOG.debug('No previous alarm within no_repeat_interval. '
                            'No previous alarm.')
             else:
                 diff = time.time() - last_alarm_time
                 _LOG.debug('No previous alarm within no_repeat_interval. Time '
                            'since last: {0:.1f}'.format(diff))
-
-            query = "INSERT INTO alarm_log (alarm_id) VALUES (%s)"
-            self._alarm_cursor.execute(query, (alarm['id']))
+            # Todo: Consider to write a prepared statement rather than using format
```

Comment:
Agreed

Reply:
However, if for the purpose of security or doing it proper, I actually **think** the way it was originally done, takes care of all the proper escaping i.e. while it may be old syntax I think it was correct and safe.

Reply:
It was correct and safe, but the syntax is deprecated, in the interest of actually getting this to run, I have chosen the fast approach for now :)

Reply:
Gotcha